app/commands/secret.py

```diff
@@ -0,0 +1,163 @@
+import i18n
+import requests
+import time
+from commands.utils import get_user_locale
+
+i18n.load_path.append("./commands/locales/")
+
+i18n.set("locale", "en-US")
+i18n.set("fallback", "en-US")
+
+
+def secret_command(client, ack, command, body):
+    ack()
+    if "user" in body:
+        user_id = body["user"]["id"]
+    else:
+        user_id = body["user_id"]
+    locale = get_user_locale(user_id, client)
+    i18n.set("locale", locale)
+    view = generate_secret_command_modal_view(command, user_id, locale)
+    client.views_open(trigger_id=body["trigger_id"], view=view)
+
+
+def secret_view_handler(ack, client, view, logger):
+    ack()
+    locale = view["blocks"][0]["elements"][0]["value"]
+    i18n.set("locale", locale)
+    secret = view["state"]["values"]["secret_input"]["secret_submission"]["value"]
+    ttl = view["state"]["values"]["product"]["secret_ttl"]["selected_option"]["value"]
+
+    # Encrypted message API
+    endpoint = "https://encrypted-message.cdssandbox.xyz/encrypt"
```

Comment:
Super, super minor thing which we don't need to change, but the URL https://encrypted-message.cdssandbox.xyz might be better to live in a variable or secret since you are anyways using it in multiple places throughout your code. Again super minor.

Reply:
Agreed - I hardcoded it for now but should be made into an env in the future